<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>124</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Key Generator</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icon.png</normaloff>../icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0" colspan="2">
     <widget class="QLineEdit" name="line_email">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="button_sign">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Sign</string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Email address:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>318</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuKey">
    <property name="title">
     <string>Key</string>
    </property>
    <addaction name="action_new_pair"/>
    <addaction name="separator"/>
    <addaction name="action_preferences"/>
    <addaction name="separator"/>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menuKey"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="action_new_pair">
   <property name="text">
    <string>New Pair</string>
   </property>
   <property name="toolTip">
    <string>New Pair</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="action_preferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>action_quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>158</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>line_email</sender>
   <signal>returnPressed()</signal>
   <receiver>button_sign</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
